Source/iOS/Utilities.swift

```diff
@@ -0,0 +1,13 @@
+import UIKit
+
+public struct Utilities {
+
+  public static var isPushNotificationEnabled: Bool {
```

Comment:
Since there are many push notifications, I think this should be pluralized to become `isPushNotificationsEnabled`